paas/smoke-test-apps.yml

```diff
@@ -34,5 +34,8 @@ applications:
       NOTIFICATIONS_URL: ((notifications_url))
       DIRECT_DEBIT_GOCARDLESS_WEBHOOK_SECRET: ((direct_debit_gocardless_webhook_secret))
 
+  - <<: *smoke-test-app
```

Comment:
Not sure you can use the anchor as-is because the app urls differ across environment, e.g. `PUBLICAPI_URL` and `NOTIFICATIONS_URL`. Maybe you could do something like the following making sure to update the env files accordingly https://github.com/alphagov/pay-omnibus/tree/master/paas/env_variables
```
NOTIFICATIONS_URL: https://notifications.((environment))
```